core/src/main/scala/wartremover/warts/Equals.scala

```diff
@@ -7,8 +7,11 @@ object Equals extends WartTraverser {
   def apply(u: WartUniverse): u.Traverser = {
     import u.universe._
 
-    val Equals: TermName = NameTransformer.encode("==")
-    val NotEquals: TermName = NameTransformer.encode("!=")
+    val Eqeq: TermName = NameTransformer.encode("==")
+    val NotEq: TermName = NameTransformer.encode("!=")
+    val Equals: TermName = "equals"
+    val Eq: TermName = "eq"
+    val Ne: TermName = "ne"
```

Comment:
IMO, they indicate low-level code (like `While`). But first of all, I followed [this](https://github.com/wartremover/wartremover/issues/347#issuecomment-293745546) suggestion. @ClaireNeveu Could you please comment?